### PR TITLE
Fixed js error when adding new group

### PR DESCRIPTION
**Pull request by skjnldsv**
There was an unneeded line on #11193 that generated an error (forgot to delete)
This is non breaking though

### CODE REVIEW

File: settings/src/views/Users.vue [hunk 1]
```diff
@@ -218,7 +218,6 @@ export default {
 				.catch(() => {
 					this.loadingAddGroup = false;
 				});
-			return this.$store.getters.getGroups[this.groups.length];
```
Comment (rullzer) [inline settings/src/views/Users.vue]: But now the docs lie right?

Comment (skjnldsv) [reply settings/src/views/Users.vue]: DAMMIT

Comment (skjnldsv) [reply settings/src/views/Users.vue]: Fixed ;)

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

**Review by rullzer — APPROVED** [review]

**Pull request merged**